lib/modules/manager/flux/extract.ts

```diff
@@ -20,6 +21,17 @@ import type {
   SystemFluxManifest,
 } from './types';
 
+export function splitTagParts(currentTag: string): PackageDependency {
```

Comment:
i would prefer to use the getDep function of dockerfile manager

Reply:
I have made those changes, I now have it using `getDep()`.